src/Swarrot/Broker/MessagePublisher/PublishConfirmPublisherInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+declare(strict_types=1);
+
+namespace Swarrot\Broker\MessagePublisher;
+
+/**
+ * Interface PublishConfirmPublisherInterface
```

Comment:
this phpdoc is totally useless. It should explain what the interface represent, not what its name is (I can already know it by reading the next line)